libs/jquery.signalR-2.0.3.js

```diff
@@ -40,6 +40,13 @@
         throw new Error(resources.nojQuery);
     }
 
+    window.$signalrStats = {
```

Comment:
Those changes will be dropped as soon as I finish profiling size and performance difference between signalR and new plain websocket.